src/services/completions.ts

```diff
@@ -1342,9 +1343,12 @@ namespace ts.Completions {
                     }
 
                     const symbolId = getSymbolId(symbol);
-                    symbols.push(symbol);
-                    symbolToOriginInfoMap[symbolId] = origin;
-                    symbolToSortTextMap[symbolId] = SortText.AutoImportSuggestions;
+                    const existingSymbol = findLast(symbols, symbol => symbol.id === symbolId);

```

Comment:
Maybe there is a better way than the linear lookup on the id of symbols? This is pretty late in the completions call, so it was about ~100 items long in this test. Didn't look like there was an easier way, though maybe I could misuse `symbolToOriginInfoMap` or `symbolToSortTextMap`?